glances_api/__init__.py

```diff
@@ -31,13 +32,16 @@ def __init__(
         self.values = None
         self.username = username
         self.password = password
+        self.custom_httpx_client = custom_httpx_client
 
     async def get_data(self, endpoint):
         """Retrieve the data."""
         url = "{}/{}".format(self.url, endpoint)
 
+        httpxClient = this.custom_httpx_client if this.custom_httpx_client else httpx.AsyncClient()
```

Comment:
Python prefers snake_case in variable names.
```suggestion
        httpx_client = self.httpx_client if self.httpx_client else httpx.AsyncClient()
```